packages/sdk/src/evm/contracts/smart-contract.ts

```diff
@@ -117,6 +120,13 @@ export class SmartContract<TContract extends BaseContract = BaseContract>
     return assertEnabled(this.detectOwnable(), FEATURE_OWNER);
   }
 
+  /**
+   * Set and get the appuri of the contract
+   */
+  get appuri(): ContractAppURI<AppURI> {
```

Comment:
Same comment here about namespace or capitalization

Reply:
Same response

Reply:
let's make this `appURI` as well then, in line with the comment above

Reply:
Fixed in latest